### PR TITLE
contributors: fix help-wanted issues link in readme

### LINKED ISSUE #530: Fix wrong help-wanted issues link
In this doc - https://github.com/pingcap/community/blob/master/contributors/README.md#your-first-contribution , the link to help-wanted issues seems to be wrong

Wrong link - https://github.com/pingcap/tidb/issues?q=is%3Aissue+is%3Aopen+label%3Astatus%2Fhelp-wanted

Right link - https://github.com/pingcap/tidb/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22

### PR DESCRIPTION
**Pull request by karuppiah7890**
closes #530

### CODE REVIEW

**Comment by karuppiah7890** [comment]
@tisonkun

File: contributors/README.md [hunk 1]
```diff
@@ -58,7 +58,7 @@ workspace](https://github.com/tikv/tikv/blob/master/CONTRIBUTING.md#building-and
 ## Your First Contribution
 
 All set to contribute? You can start by finding an existing issue with the
-[help-wanted](https://github.com/pingcap/tidb/issues?q=is%3Aissue+is%3Aopen+label%3Astatus%2Fhelp-wanted) label in the tidb repository. These issues are well suited for new contributors.
+[help-wanted](https://github.com/pingcap/tidb/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22) label in the tidb repository. These issues are well suited for new contributors.
```
Comment (tisonkun) [inline contributors/README.md:61]:
```suggestion
[help wanted](https://github.com/pingcap/tidb/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22) label in the tidb repository. These issues are well suited for new contributors.
```

to be accurate 😄

**Review by tisonkun — APPROVED** [review]

**Comment by tisonkun** [comment]
/merge

Thanks for your contribution @karuppiah7890 !

File: contributors/README.md (hunk 1 above)
Comment (karuppiah7890) [reply contributors/README.md]: Oops. Just noticed this

Comment (karuppiah7890) [reply contributors/README.md]: Thanks for updating it! 🙏

**Pull request merged**